File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/MailRow.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// A result row from the &#x60;Mail&#x60; &#x60;GET&#x60; request.
  /// </summary>
  [DataContract]
  public class MailRow {
    /// <summary>
    /// The id of the mail.
    /// </summary>
    /// <value>The id of the mail.</value>
    [DataMember(Name="mail_id", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "mail_id")]
    public string MailId { get; set; }

    /// <summary>
    /// The repeat invoices cost of the mail.
    /// </summary>
    /// <value>The repeat invoices cost of the mail.</value>
    [DataMember(Name="repeat_invoices_cost", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "repeat_invoices_cost")]
    public string RepeatInvoicesCost { get; set; }

    /// <summary>
    /// The username of the mail.
    /// </summary>
    /// <value>The username of the mail.</value>
    [DataMember(Name="mail_username", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "mail_username")]
    public string MailUsername { get; set; }

    /// <summary>
    /// The status of the mail.
    /// </summary>
    /// <value>The status of the mail.</value>
    [DataMember(Name="mail_status", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "mail_status")]
    public string MailStatus { get; set; }

    /// <summary>
    /// The services name of the mail.
    /// </summary>
    /// <value>The services name of the mail.</value>
    [DataMember(Name="services_name", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "services_name")]
    public string ServicesName { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class MailRow {\n");
      sb.Append("  MailId: ").Append(MailId).Append("\n");
      sb.Append("  RepeatInvoicesCost: ").Append(RepeatInvoicesCost).Append("\n");
      sb.Append("  MailUsername: ").Append(MailUsername).Append("\n");
      sb.Append("  MailStatus: ").Append(MailStatus).Append("\n");
      sb.Append("  ServicesName: ").Append(ServicesName).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/mail_row.dart
part of swagger.api;

class MailRow {
  /* The id of the mail. */
  String mailId = null;
/* The repeat invoices cost of the mail. */
  String repeatInvoicesCost = null;
/* The username of the mail. */
  String mailUsername = null;
/* The status of the mail. */
  String mailStatus = null;
/* The services name of the mail. */
  String servicesName = null;

  MailRow();

  @override
  String toString() {
    return 'MailRow[mailId=$mailId, repeatInvoicesCost=$repeatInvoicesCost, mailUsername=$mailUsername, mailStatus=$mailStatus, servicesName=$servicesName, ]';
  }

  MailRow.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    mailId = json['mail_id'];
    repeatInvoicesCost = json['repeat_invoices_cost'];
    mailUsername = json['mail_username'];
    mailStatus = json['mail_status'];
    servicesName = json['services_name'];
  }

  Map<String, dynamic> toJson() {
    return {
      'mail_id': mailId,
      'repeat_invoices_cost': repeatInvoicesCost,
      'mail_username': mailUsername,
      'mail_status': mailStatus,
      'services_name': servicesName
     };
  }

  static List<MailRow> listFromJson(List<dynamic> json) {
    return json == null ? new List<MailRow>() : json.map((value) => new MailRow.fromJson(value)).toList();
  }

  static Map<String, MailRow> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, MailRow>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new MailRow.fromJson(value));
    }
    return map;
  }
}
